content/tutorial-build-run-rippled-ubuntu.md

```diff
@@ -0,0 +1,165 @@
+
+6. Install `pkg-config`.
+
+        sudo apt-get -y install pkg-config
+
+7. Install Protocol Buffers.
+
+        sudo apt-get -y install protobuf-compiler
+        sudo apt-get -y install libprotobuf-dev
+
+8. Install Secure Socket Layer (SSL) toolkit development files.
+
+        sudo apt-get -y install libssl-dev
+
+9. Install `python-software-properties`.
+
+        sudo apt-get install -y python-software-properties
+
+10. Install Boost.
+
+        sudo add-apt-repository -y ppa:boost-latest/ppa
+        sudo apt-get -y update
+        sudo apt-get install -y libboost-all-dev
+
+11. Get `rippled` source code.
+
+        git clone https://github.com/ripple/rippled.git
+        cd rippled
+        git checkout master
+
+12. Build `rippled` binary executable from source code. This may take about 30 minutes, depending on your hardware specs.
+
+        scons
+
+    SCons saves the built executable in `rippled/build`.
+
+
+## 2. Configure `rippled`
+
+Complete the following configurations that are required for `rippled` to start up successfully. All other configuration is optional and can be tweaked after you have a working server.
+
+1. Create a copy of the example config file (assumes you're in the `rippled` folder already). Saving the config file to this location enables you to run `rippled` as a non-root user (recommended).
+
+         cp doc/rippled-example.cfg ~/.config/ripple/rippled.cfg
+
+2. Edit the config file to set necessary file paths. The user you plan to run `rippled` as must have write permissions to all of the paths you specify here.
+
+    1. Set the `[node_db]`'s path to the location where you want to store the ledger database.
+
+    2. Set the `[database_path]` to the location where you want to store other database data. (This includes an SQLite database with configuration data, and is typically one level above the `[node_db]` path field.)
+
+    3. Set the `[debug_logfile]` to a path where `rippled` can write logging information.
+
+3. Copy the example `validators.txt` file to the same folder as `rippled.cfg`:
+
+        cp doc/validators-example.txt ~/.config/ripple/validators.txt
+
+    **Warning:** Ripple has designed and [documented](https://ripple.com/dev-blog/decentralization-strategy-update/) a decentralization plan with maximum safety in mind. During the transition, you *should not* modify the `validators.txt` file except as recommended by Ripple. Even minor modifications to your validator settings could cause your server to diverge from the rest of the network and report out of date, incomplete, or inaccurate data. Acting on such data can cause you to lose money.
+
+## 3. Run `rippled`
+
+To run your stock `rippled` server from the executable you built, using the configurations you defined:
+```
+cd build
+./rippled
+```
+
+
+### What to Expect
+
+Once you've run `rippled`, here are excerpts of what you can expect to see in your terminal.
+
+```
+Loading: "/etc/opt/ripple/rippled.cfg"
```

Comment:
This path shouldn't exist if following this guide.

Also /etc/opt/ripple/rippled.cfg is not the correct location for a config file on Debian based systems.

Reply:
Yeah, I'm not sure what's up with that. I think it's supposed to use an XDG_* environment variable. But also if you run `rippled` _not_ as root, it should look in `~/.config/ripple/rippled.cfg` first.